app/src/main/java/net/gsantner/markor/activity/StoragePermissionActivity.java

```diff
@@ -0,0 +1,69 @@
+package net.gsantner.markor.activity;
```

Comment:
Is that activity used somewhere? I checked the PR and didn't see any startActivity or alike for this one. It's from a previous fix experiment right